src/MPDevice.h

```diff
@@ -96,7 +98,7 @@ class MPDevice: public QObject
     QT_WRITABLE_PROPERTY(qint64, uid, -1)
 
 public:
-    MPDevice(QObject *parent);
+    MPDevice(QObject *parent, bool isBLE = false);
```

Comment:
I don't like so much this boolean that is passed all over constuctor... There is already a `bool isMiniFlag = false;` for the variant of Mooltipass (first edition the big one), the Mini, and now the BLE. It would be better to have some kind of enum somewhere to keep the device type. Booleans spreads all over the code are not so good for maintenance...

Reply:
Created an enum for device type, but needed to move the message protocol init from base class constructor.